### PR TITLE
Surf 848 typography tests

### PR DESCRIPTION
**Pull request by Droogans**
JIRA: https://jira.rax.io/browse/SURF-848

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

### CODE REVIEW

File: docs/components/typography/index.html [hunk 1]
```diff
@@ -34,6 +35,7 @@ <h3>Heading Three</h3>
       <h4>Heading Four</h4>
       <h5>Heading Five</h5>
     </div>
+  </section>
```
Comment (CITguy) [inline docs/components/typography/index.html]: This creates invalid markup. You've closed sections without opening new ones. To remediate:

1. Revert the hierarchy changes
2. Add `[data-visreg]` attributes to `div.demo` elements instead

File: docs/components/typography/index.html [hunk 2]
```diff
@@ -199,14 +207,19 @@ <h2 id="keyboard">Keyboard</h2>
...
   </section>
 
-  <section><!-- Code Block -->
+  <section data-visreg="typography-code-block"><!-- Code Block -->
     <h2 id="code-block">Code Block</h2>
+    <p>Example (python):</p>
 
     <div class="demo">
-      <pre><code>[Insert Code Here]</code></pre>
+      <pre><code>>>> print round(4 * sum(-1.0 ** (n%2) / (2*n + 1) for n in range(9999999)), 6))
```
Comment (CITguy) [inline docs/components/typography/index.html]: Please revert these changes and create an appropriate `<figure>` below alongside the other examples.

File: docs/components/typography/index.html [hunk 3]
```diff
@@ -130,14 +137,15 @@ <h2 id="sub-body">Sub-Body</h2>
     <div class="demo">
       <p class="hxSubBody">Sub-Body</p>
     </div>
+  </section>
```
Comment (CITguy) [inline docs/components/typography/index.html]: This creates invalid markup. See comment on line 38.

File: docs/components/typography/index.html [hunk 4]
```diff
@@ -106,6 +111,7 @@ <h2 id="caption">Caption</h2>
     <div class="demo">
       <p class="hxCaption">Caption</p>
     </div>
+  </section>
```
Comment (CITguy) [inline docs/components/typography/index.html]: This creates invalid markup. See comment on line 38.

File: docs/components/typography/index.html [hunk 5]
```diff
@@ -86,6 +89,7 @@ <h2 id="subdued">Subdued</h2>
     <div class="demo">
       <span class="hxSubdued">Subdued</span>
     </div>
+  </section>
```
Comment (CITguy) [inline docs/components/typography/index.html]: This creates invalid markup. See comment on line 38.

File: docs/components/typography/index.html (hunk 1 above)
Comment (Droogans) [reply docs/components/typography/index.html]: I see so you can't nest `section` tags like this? Interesting.

I didn't want to capture the output below the visual examples, such as description text. Seemed like that wasn't as useful as capturing just the content of the rendered demo part, not the entire demo itself.

Comment (CITguy) [reply docs/components/typography/index.html]: If you target the `<div>` elements with the `.demo` class, it should avoid the code snippets.

**Comment by Droogans** [comment]
@CITguy Updated with fixes from feedback.
